relay/relay_test.go

```diff
@@ -0,0 +1,36 @@
+package relay_test
+
+import (
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/neelance/graphql-go"
+	"github.com/neelance/graphql-go/example/starwars"
+	"github.com/neelance/graphql-go/relay"
+)
+
+var starwarsSchema = graphql.MustParseSchema(starwars.Schema, &starwars.Resolver{})
+
+func TestServeHTTP(t *testing.T) {
+	w := httptest.NewRecorder()
+	r := httptest.NewRequest("POST", "/some/path/here", strings.NewReader(`{"query":"{ hero { name } }", "operationName":"", "variables": null}`))
+	h := relay.Handler{Schema: starwarsSchema}
+
+	h.ServeHTTP(w, r)
+
+	if w.Code != 200 {
+		t.Fatalf("Expected status code 200, got %d.", w.Code)
+	}
+
+	contentType := w.Header().Get("Content-Type")
+	if contentType != "application/json" {
+		t.Fatalf("Invalid content-type. Expected [application/json], but instead got [%s]", contentType)
+	}
+
+	expectedResponse := `{"data":{"hero":{"name":"R2-D2"}}}`
+	actualResponse := string(w.Body.Bytes())
```

Comment:
@pavelnikolov, `string(w.Body.Bytes())` could be simplified to `w.Body.String()`.